TesseractCore/Exporting.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Exporting.swift
+//  TesseractCore
+//
+//  Created by Alexander Mackworth on 2/4/15.
+//  Copyright (c) 2015 Rob Rix. All rights reserved.
```

Comment:
Could you use the same header format as the other files? (Basically the copyright line, none of the others.)